lucille-core/src/test/resources/ConfigValidationTest/duplicate-pipeline.conf

```diff
@@ -0,0 +1,42 @@
+connectors: [
+  {
+    name: "connector1",
+    class: "com.kmwllc.lucille.connector.CSVConnector",
+    pipeline: "pipeline1",
+    idField: "id",
+    path: "classpath:RunnerTest/singleDoc.csv",
+  }
+  {
+    name: "connector2",
+    class: "com.kmwllc.lucille.connector.CSVConnector",
+    pipeline: "pipeline1",
+    idField: "id",
+    path: "classpath:RunnerTest/singleDoc.csv",
+  }
+]
+
+pipelines: [
```

Comment:
comment about the filename "duplicate-pipeline.conf"
this is really an example of a "reused" pipeline -- i.e. a pipeline that is used or shared across multiple connectors
"duplicate pipeline" suggests two pipelines definitions with the same name